<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="7" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-11-11T15:39:04 IST" name="Suite" finished-at="2024-11-11T15:39:37 IST" duration-ms="33537">
    <groups>
    </groups>
    <test started-at="2024-11-11T15:39:04 IST" name="Miscellenous Test" finished-at="2024-11-11T15:39:29 IST" duration-ms="25783">
      <class name="TestFolder.Test1">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:04 IST" name="setup" finished-at="2024-11-11T15:39:09 IST" duration-ms="5279" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="checkTheLocators()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:09 IST" name="checkTheLocators" finished-at="2024-11-11T15:39:17 IST" duration-ms="7391" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTheLocators -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:17 IST" name="tearDown" finished-at="2024-11-11T15:39:17 IST" duration-ms="562" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:17 IST" name="setup" finished-at="2024-11-11T15:39:20 IST" duration-ms="2584" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="foundThePassword()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:20 IST" name="foundThePassword" finished-at="2024-11-11T15:39:24 IST" duration-ms="4493" status="PASS">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Hello rahul,] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Hello rahul,] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at TestFolder.Test1.foundThePassword(Test1.java:123)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- foundThePassword -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:24 IST" name="tearDown" finished-at="2024-11-11T15:39:25 IST" duration-ms="507" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:25 IST" name="setup" finished-at="2024-11-11T15:39:27 IST" duration-ms="2082" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="openGoogleTest()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:27 IST" name="openGoogleTest" finished-at="2024-11-11T15:39:29 IST" duration-ms="1967" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openGoogleTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestFolder.Test1@3a3e78f]" started-at="2024-11-11T15:39:29 IST" name="tearDown" finished-at="2024-11-11T15:39:29 IST" duration-ms="500" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestFolder.Test1 -->
    </test> <!-- Miscellenous Test -->
    <test started-at="2024-11-11T15:39:04 IST" name="Error Validation Test" finished-at="2024-11-11T15:39:33 IST" duration-ms="29170">
      <class name="TestFolder.ErrorValidationTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestFolder.ErrorValidationTest@1f6c9cd8]" started-at="2024-11-11T15:39:04 IST" name="setUp" finished-at="2024-11-11T15:39:12 IST" duration-ms="8031" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginErrorValiadtionTest()[pri:0, instance:TestFolder.ErrorValidationTest@1f6c9cd8]" started-at="2024-11-11T15:39:12 IST" name="loginErrorValiadtionTest" finished-at="2024-11-11T15:39:12 IST" duration-ms="780" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValiadtionTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestFolder.ErrorValidationTest@1f6c9cd8]" started-at="2024-11-11T15:39:12 IST" name="tearDown" finished-at="2024-11-11T15:39:13 IST" duration-ms="693" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestFolder.ErrorValidationTest@1f6c9cd8]" started-at="2024-11-11T15:39:13 IST" name="setUp" finished-at="2024-11-11T15:39:16 IST" duration-ms="3238" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="productErrorvalidationTest()[pri:0, instance:TestFolder.ErrorValidationTest@1f6c9cd8]" started-at="2024-11-11T15:39:16 IST" name="productErrorvalidationTest" finished-at="2024-11-11T15:39:32 IST" duration-ms="15579" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productErrorvalidationTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestFolder.ErrorValidationTest@1f6c9cd8]" started-at="2024-11-11T15:39:32 IST" name="tearDown" finished-at="2024-11-11T15:39:33 IST" duration-ms="691" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestFolder.ErrorValidationTest -->
    </test> <!-- Error Validation Test -->
    <test started-at="2024-11-11T15:39:04 IST" name="Stand Alone Test" finished-at="2024-11-11T15:39:37 IST" duration-ms="33537">
      <class name="TestFolder.StandAloneTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestFolder.StandAloneTest@736caf7a]" started-at="2024-11-11T15:39:04 IST" name="setUp" finished-at="2024-11-11T15:39:11 IST" duration-ms="7170" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="endToEndTest()[pri:0, instance:TestFolder.StandAloneTest@736caf7a]" started-at="2024-11-11T15:39:11 IST" name="endToEndTest" finished-at="2024-11-11T15:39:15 IST" duration-ms="4409" status="PASS">
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[element click intercepted: Element <button _ngcontent-wue-c36="" type="button" class="btn btn-primary">...</button> is not clickable at point (1039, 473). Other element would receive the click: <div class="ng-tns-c4-2 ng-star-inserted ng-trigger ng-trigger-flyInOut ngx-toastr toast-success" toast-component="" style="opacity: 1;">...</div>
  (Session info: chrome=130.0.6723.116)
Build info: version: '4.5.3', revision: '4b786a1e430'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.11'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [5879b2caef3ca9fa976cbae02f046f40, clickElement {id=f.FEAAE537D3D8A936FFDB0405A1EEC2DA.d.772B10AE20164F331633CF2BD44C4517.e.103}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 130.0.6723.116, chrome: {chromedriverVersion: 130.0.6723.116 (6ac35f94ae3..., userDataDir: C:\Users\RAVI~2.AGG\AppData...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:54874}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:54874/devtoo..., se:cdpVersion: 130.0.6723.116, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (5879b2caef3ca9fa976cbae02f046f40)] -> xpath: //button[contains(text(),'Checkout')]]
Session ID: 5879b2caef3ca9fa976cbae02f046f40]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <button _ngcontent-wue-c36="" type="button" class="btn btn-primary">...</button> is not clickable at point (1039, 473). Other element would receive the click: <div class="ng-tns-c4-2 ng-star-inserted ng-trigger ng-trigger-flyInOut ngx-toastr toast-success" toast-component="" style="opacity: 1;">...</div>
  (Session info: chrome=130.0.6723.116)
Build info: version: '4.5.3', revision: '4b786a1e430'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.11'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [5879b2caef3ca9fa976cbae02f046f40, clickElement {id=f.FEAAE537D3D8A936FFDB0405A1EEC2DA.d.772B10AE20164F331633CF2BD44C4517.e.103}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 130.0.6723.116, chrome: {chromedriverVersion: 130.0.6723.116 (6ac35f94ae3..., userDataDir: C:\Users\RAVI~2.AGG\AppData...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:54874}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:54874/devtoo..., se:cdpVersion: 130.0.6723.116, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (5879b2caef3ca9fa976cbae02f046f40)] -> xpath: //button[contains(text(),'Checkout')]]
Session ID: 5879b2caef3ca9fa976cbae02f046f40
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:541)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:257)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:78)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at jdk.proxy2/jdk.proxy2.$Proxy23.click(Unknown Source)
at GreenCart.PageObject.CartPage.goToCheckOut(CartPage.java:40)
at TestFolder.StandAloneTest.endToEndTest(StandAloneTest.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endToEndTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestFolder.StandAloneTest@736caf7a]" started-at="2024-11-11T15:39:16 IST" name="tearDown" finished-at="2024-11-11T15:39:16 IST" duration-ms="495" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestFolder.StandAloneTest@736caf7a]" started-at="2024-11-11T15:39:16 IST" name="setUp" finished-at="2024-11-11T15:39:20 IST" duration-ms="3679" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="orderHistoryTest()[pri:0, instance:TestFolder.StandAloneTest@736caf7a]" started-at="2024-11-11T15:39:20 IST" name="orderHistoryTest" finished-at="2024-11-11T15:39:37 IST" duration-ms="16683" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestFolder.StandAloneTest@736caf7a]" started-at="2024-11-11T15:39:37 IST" name="tearDown" finished-at="2024-11-11T15:39:37 IST" duration-ms="488" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestFolder.StandAloneTest -->
    </test> <!-- Stand Alone Test -->
  </suite> <!-- Suite -->
</testng-results>
